api/v1alpha1/capp_types.go

```diff
@@ -138,13 +134,13 @@ type RevisionInfo struct {
 	RevisionName string `json:"name,omitempty"`
 }
 
-type EnabledStatus struct {
-	// IsEnabled is actual enabled state of the capp
-	// +kubebuilder:default:=false
+type StateStatus struct {
+	// State is actual enabled state of the capp
+	// +kubebuilder:default:="running"
 	// +optional
-	IsEnabled bool `json:"isEnabled,omitempty"`
+	State string `json:"state,omitempty"`
 
-	// LastEnabledChange the last time the enabled state of capp changed
+	// LastChange the last time the enabled state of capp changed
```

Comment:
// LastChange is the last time the enabled state of Capp changed

Reply:
i also removed the enabled